spacy_experimental/coref/tests/test_coref.py

```diff
@@ -7,6 +7,7 @@
 from spacy.tests.util import make_tempdir
 import spacy_experimental.coref
 from spacy_experimental.coref.coref_util import (
+    DEFAULT_CLUSTER_HEAD_PREFIX,
     DEFAULT_CLUSTER_PREFIX,
     select_non_crossing_spans,
     get_sentence_ids,
```

Comment:
I understand this syntax should be avoided instead of multiple `from ...` lines, but when I refactor that here I can only fit about one variable per line, so I think this is the lesser evil here.

Reply:
Nevermind, I forgot I should use relative imports here.

Reply:
No, use absolute imports in the test suite.

Reply:
Ah, right. Reverted.